setup.py

```diff
@@ -16,7 +16,7 @@
 from setuptools import setup, find_packages
 
 
-VERSION = "0.23.0"
+VERSION = "0.22.3"
```

Comment:
is this correct? we are on. 0.22.3 currently confused why conflict is with 0.23

Reply:
The `develop` branch is 0.23 (the "next" minor release)
The `release` branch is 0.22.3 (the "next" patch release)

you should keep 0.23.0 on `develop`!